### ISSUE TITLE
Is there a method to export obj?

### ISSUE THREAD

**Issue by wells-wei-wei** [ahmedosman/STAR#2]

**Comment by ahmedosman** [comment]
You can use any rendering library (for example trimesh , MPI mesh package) to export the output as an  obj. 

For example:

d = star(poses, betas, trans) 

mesh_vertices=d.detach().cpu().numpy().squeeze()

mesh_faces=d.f

out_mesh = trimesh.Trimesh(mesh_vertices, mesh_faces)
